src/Sarif/Readers/PropertyBagConverter.cs

```diff
@@ -106,5 +81,37 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 
             writer.WriteEndObject();
         }
+
+        public static JTokenType DotNetTypeToJTokenType(Type dotNetType)
+        {
+            JTokenType jTokenType = JTokenType.None;
```

Comment:
Having factored this out so that it could also be used by `PropertyBagHolder.SetProperty` (see above), I now see that it doesn't handle the case where `SetProperty` is called with anything other than one of the Json.NET types -- it doesn't handle an arbitrary class. So I have more work to do here and in `SetProperty`. You can defer finishing the review until I've addressed this.